ccm_web/client/src/components/CreateSelectSectionWidget.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
```

Comment:
Really nice that you separated this as component and can be reused for future features!